code/src/components/ThoughtsList.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import moment from 'moment';
+
+import './Styles/thoughtslist.css';
+
+export const ThoughtsList = ({thought, onLike}) => {
```

Comment:
Maybe reconsider naming of the component, since this is in fact a Thought and not the list? :)